docs/zh_CN/windowsofflineinstaller.rst

```diff
@@ -1 +1,130 @@
-.. include:: ../en/windowsofflineinstaller.rst
+.. _windowsofflineinstaller:
```

Comment:
Same here. We can remove this entire page and it's references as we don't use Windows offline installer with the v4.0.0, we will be using EIM for installation and managing the esp-idf versions.